internal/service/api/router.go

```diff
@@ -36,41 +35,37 @@ func (r *router) apiRouter() chi.Router {
 	)
 
 	router.Route("/integrations/slack", func(r chi.Router) {
-		// r.With(auth.Jwt(secret, data.ModuleName, []string{data.Roles[data.Admin], data.Roles[data.Owner], data.Roles[data.Member]}...)).
-		// 	Get("/get_input", handlers.GetInputs)
-		// r.With(auth.Jwt(secret, data.ModuleName, []string{data.Roles[data.Admin], data.Roles[data.Owner], data.Roles[data.Member]}...)).
-		// 	Get("/get_available_roles", handlers.GetRoles)
+		r.With(auth.Jwt(secret, data.ModuleName, []string{data.Roles[data.Admin], data.Roles[data.Owner], data.Roles[data.Member]}...)).
```

Comment:
I've just come up with idea to wrap this in local middleware, in order to have just `middlewares.IsAdmin`, `middlewares.IsAuthenticated`. What do you think? I reckon  that it can look cleaner

Reply:
Also If it's possible you may use some methods like r.Group, r.Use in order to apply middlewares to several routes by one line, but it is not always possible